GitExtUtils/GitUI/IToolStripEx.cs

```diff
@@ -3,7 +3,7 @@
     public interface IToolStripEx
     {
         /// <summary>
-        /// Gets or sets whether the toolstrip control has a border drawn around it.
+        /// Gets or sets whether the tool strip control has a border drawn around it.
```

Comment:
I find toolstrip to be a name here

Reply:
The name is `ToolStrip` which is two words. Also `MenuStrip`, `StatusStrip`, etc.

https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.toolstrip?view=netframework-4.8

Reply:
It's common to refer to those as "toolstrip", "menustrip", etc. Like "textbox" vs "text box".